.gitignore

```diff
@@ -30,3 +30,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### Docker ###
+.env
```

Comment:
.env file in pet-project might not ignore for other people to easily test your project, but it is arguably